project/src/components/app/app.tsx

```diff
@@ -14,12 +24,40 @@ function App({
   moviesList,
 }: AppMovieProps): JSX.Element {
   return (
-    <Main
-      movieTitle={movieTitle}
-      movieGenre={movieGenre}
-      movieReleaseDate={movieReleaseDate}
-      moviesList={moviesList}
-    />
+    <BrowserRouter>
+      <Switch>
+        <Route exact path={AppRoutes.Main}>
+          <Main
+            movieTitle={movieTitle}
+            movieGenre={movieGenre}
+            movieReleaseDate={movieReleaseDate}
+            moviesList={moviesList}
+          />
+        </Route>
+        <Route exact path={AppRoutes.SignIn}>
+          <SignIn />
+        </Route>
+        <PrivateRoute
+          exact
+          path={AppRoutes.MyList}
+          render={() => <MyList />}
+          authStatus={AuthStatus.NoAuth}
+        >
+        </PrivateRoute>
+        <Route exact path={AppRoutes.Movie}>
+          <Movie />
+        </Route>
+        <Route exact path={AppRoutes.AddReview}>
+          <AddReview />
+        </Route>
+        <Route exact path={AppRoutes.Player}>
+          <Player />
+        </Route>
+        <Route>
```

Comment:
Давай укажем явно что любой путь, так же важно следить за порядком подключения здесь, если ты поднимешь 404 выше то он будет матчить все пути и всегда рисоваться
```suggestion
        <Route path="*">
```